packages/apps/patron-deletion-tracker/src/app.ts

```diff
@@ -1,5 +1,60 @@
+import { SierraClient } from '@weco/sierra-client';
+import { Auth0Client } from '@weco/auth0-client';
+import { ResponseStatus } from '@weco/identity-common';
 import { ScheduledHandler } from 'aws-lambda';
+import { startOfYesterday, endOfYesterday } from 'date-fns';
+import { rateLimit } from './rate-limits';
 
-export const lambdaHandler: ScheduledHandler = async (event) => {
-  console.log('I am a Patron deletion tracker');
+const auth0RateLimited = rateLimit({
+  // See https://auth0.com/docs/troubleshoot/customer-support/operational-policies/rate-limit-policy/management-api-endpoint-rate-limits#self-service-subscription-limits
```

Comment:
It'd be helpful to explain which endpoint we're using – although I can see it's "Write users" now because it's the only endpoint with these numbers, if those change it'll be harder to tell.

Also, is max concurrency set by us or by Auth0?